doc/lua_api.md

```diff
@@ -6922,6 +6922,15 @@ Timing
     * Optional: Variable number of arguments that are passed to `func`
     * Jobs set for earlier times are executed earlier. If multiple jobs expire
       at exactly the same time, then they are executed in registration order.
+    * `time` is a lower bound. The job is executed in the first server-step that
+      started at least `time` seconds after the last time a server-step started,
+      measured with globalstep dtime.
+    * In particular this can result in relatively large delays if `time` is close
+      to the server-step dtime. For example, with a target server-step of 0.09 s
+      (the engine uses the target as a lower bound for the actual server-step
+      dtime), `minetest.after(0.09, ...)` usually waits two steps, resulting in
+      a delay of about 0.18 s.
```

Comment:
> the engine uses the target as a lower bound for the actual server-step dtime

This means that if I have `dedicated_server_step = 0.09`, that value is supposed to be used as a lower bound, so the actual dtime is supposed to be at least 0.09, correct?

> `minetest.after(0.09, ...)` usually waits two steps, resulting in a delay of about 0.18 s.

This thus contradicts the previous statement. If the actual dtime was always at least 0.09, it would never wait two steps.

Reply:
Oh, I see you already fixed that.

However, this:

>  the engine uses the target as a lower bound for the actual server-step dtime

is indeed the behavior I would expect. It would fix Zughy's problem and it would be consistent with `core.after`, where the given time is also a lower bound. See https://github.com/minetest/minetest/issues/15193#issuecomment-2435160348 :D

Reply:
Yeah, I got a bit confusion there. x)
Latest version should be correct.